src/test/java/net/sf/jabref/logic/autosaveandbackup/BackupManagerTest.java

```diff
@@ -0,0 +1,18 @@
+package net.sf.jabref.logic.autosaveandbackup;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BackupManagerTest {
+
+    @Test
+    public void testBackupFileNameGenerationWithLinuxPath() {
```

Comment:
Please follow the naming convention for tests as outlined in the wiki 😸 
That is, no ̀ test` prefix and the test name should state the desired outcome (i.e. getBackupPathAppendsSavExtension)